### PR TITLE
Don't use std::unique_ptr for SVEvent

### PR DESCRIPTION
**Pull request by rmast**
https://github.com/tesseract-ocr/tesseract/issues/3869

### CODE REVIEW

**Comment by stweil** [comment]
This reverts commit 37b33749d, but misses some destructor calls which were previously used.

File: src/viewer/scrollview.cpp [hunk 1]
```diff
@@ -836,7 +836,7 @@ char ScrollView::Wait() {
   char ret = '\0';
   SVEventType ev_type = SVET_ANY;
   do {
-    std::unique_ptr<SVEvent> ev(AwaitEvent(SVET_ANY));
+    SVEvent *ev(AwaitEvent(SVET_ANY));
```
Comment (stweil) [inline src/viewer/scrollview.cpp:839]: 4.0.0 already used `std::unique_ptr` for this functionality.

**Review by stweil — CHANGES_REQUESTED** [review]

**Comment by rmast** [comment]
I didn’t test against 4.0.0.

4.1.3:

***@***.***:~/tesseract4.1.3$ grep unique_ptr src/viewer/scrollview.cpp
***@***.***:~/tesseract4.1.3$

**Comment by rmast** [comment]
Conclusion: Also /src/cutil/callcpp.cpp contains a unique_ptr-construction, but that’s probably not getting ‘delete’d. The big log is not falling over it.

I think a clear choice has to be made between using those std::-constructs and make the Java-application use them as well, or just remove them.

**Comment by stweil** [comment]
For 4.x that code was in callcpp.cpp (see link in my previous comment).

**Comment by rmast** [comment]
> For 4.x that code was in callcpp.cpp (see link in my previous comment).

Yes. I saw that on second look. My commit primarily reverses your second commit, but hardly the first one. I wonder whether those other variables are also transferred to and from Java, and whether the std:: -wrapping gives trouble with the Java-application.

**Pull request closed without merging**